packages/studio/src/components/ErrorComponentPreview.tsx

```diff
@@ -22,11 +31,38 @@ export default function ErrorComponentPreview(props: {
     setTooltipOpen(false);
   }, []);
 
-  const rect = containerRef.current?.getBoundingClientRect();
-  const tooltipPosition = rect && {
-    x: (rect.right + rect.left) / 2,
-    y: rect.top - 40,
-  };
+  useEffect(() => {
+    document
+      .querySelector("iframe")
+      ?.contentDocument?.addEventListener("scroll", () =>
```

Comment:
nit: can the callback for the listener be pulled out into a function so we can reuse it for adding and removing both listeners?